server/app/services/program/ProgramServiceImpl.java

```diff
@@ -600,7 +625,13 @@ public ErrorAnd<ProgramDefinition, CiviFormError> editStatus(
         && statusNameToIndex.containsKey(editedStatus.statusText())) {
       throw new DuplicateStatusException(editedStatus.statusText());
     }
+    if (editedStatus.defaultStatus().isPresent()
+        && editedStatus.defaultStatus().get().equals(true)) {
```

Comment:
**nitpick:** same as above, could use `map` to avoid the `&&`

Reply:
`editedStatus.defaultStatus().orElse(false)` is maybe even shorter?

Reply:
LGTM